osf/models/quickfiles.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import unicode_literals
+
+import logging
+
+from osf.models.node import AbstractNode, AbstractNodeManager
+from website.exceptions import NodeStateError
+
+
+logger = logging.getLogger(__name__)
+
+
+class QuickFilesManager(AbstractNodeManager):
+
+    def create_for_user(self, user):
+        possessive_title = get_quickfiles_project_title(user)
+
+        quickfiles, created = QuickFiles.objects.get_or_create(
+            title=possessive_title,
+            creator=user
+        )
+
+        if not created:
+            raise NodeStateError('Users may only have one quickfiles project')
+
+        quickfiles.add_addon('osfstorage', auth=None, log=False)
+
+        return quickfiles
+
+    def get_for_user(self, user):
+        return QuickFiles.objects.get(creator=user)
+
+
+class QuickFiles(AbstractNode):
```

Comment:
@sloria happy to do this -- just want to clarify as originally I did have `QuickFilesNode` and was asked to change it [earlier in the process](https://github.com/CenterForOpenScience/osf.io/pull/7339#discussion_r123606542) both in the model name and in methods.  Should I change back all of the Model name and all methods as well?